### ISSUE TITLE
Default value syntax

### ISSUE THREAD

**Issue by MadLittleMods** [postcss/postcss-mixins#10]
The JS function mixins work great but it would be handy to allow default values when defining a mixin.

Maybe a comma separated, `$var: value` type syntax.

```
@define-mixin icon $network: twitter, $color: blue {
    .icon.is-$(network) {
        color: $color;
    }
    .icon.is-$(network):hover {
        color: white;
        background: $color;
    }
}

@mixin icon;
@mixin icon twitter blue;
@mixin icon youtube red;
```

**Comment by ai** [comment]
Unfortunately you miss many important questions about API. `@define-mixin` with comma and spcae syntax is a bad solution. We need to select only one.

And if `@define-mixin` uses comma, so and `@mixin` should use it too.

But we can’t broke API. We need throw a warnings for a next version. And only in next major release remove space separation.

**Comment by ai** [comment]
I moved you commits to `defaults` branch. I will release default values after I will add new syntax and warnings for old syntax.

**Comment by MadLittleMods** [comment]
@ai What is wrong with supporting both spaces and comma separated? If someone has a conflicting situation they should just use commas.

I do agree that `@mixin` should support it as well though.

**Comment by ai** [comment]
Two syntaxes will make choice unmaintable in future. Stylus has good example of too complicated parser

**Comment by MadLittleMods** [comment]
While not a big deal now, I do see your point on the future maintenance cost. Looking forward to this being in a release :grinning:

**Comment by ai** [comment]
Released in 0.3.
